src/test/java/ServletUtilityTest.java

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import com.google.sps.utility.ServletUtility;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+
+/** Test Authentication Utility functions */
+@RunWith(JUnit4.class)
+public final class ServletUtilityTest {
+
+  private static HttpServletRequest request;
```

Comment:
Nit: Given that the tests are non-static, I think it'd be better for this to be non-static as well. JUnit could, in theory, create several instances of `ServletUtilityTest` and interact with them in parallel.

Reply:
Resolved! Thanks for the suggestion